src/panels/lovelace/editor/card-editor/hui-edit-card.ts

```diff
@@ -86,6 +85,19 @@ export class HuiEditCard extends LitElement {
     return this.shadowRoot!.querySelector("hui-card-preview")!;
   }
 
+  private _cardYaml: string | undefined;
+
+  private get cardYaml(): string | undefined {
```

Comment:
rename this to `_cardYaml` and the private private one to `__cardYaml`, to ensure it doesn't get set accidentally.

Reply:
I tried. The linter doesn't think __cardYaml is a good variable name.

Reply:
`// tslint:disable-next-line`. Let's not let linters rule sensible naming 👍

Reply:
Thank you!

(always relevant: https://www.youtube.com/watch?v=wf-BqAjZb8M)